### PR TITLE
dev/core#3100 - Remove grant leftovers from core xml. Remove empty grant reports menu

### PR DESCRIPTION
**Pull request by colemanw**
This ports #22893 from @demeritcowboy to 5.48

### CODE REVIEW

File: CRM/Upgrade/Incremental/sql/5.47.1.mysql.tpl [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{* file to handle db changes in 5.47.1 during upgrade *}
```
Comment (seamuslee001) [inline CRM/Upgrade/Incremental/sql/5.47.1.mysql.tpl:1]: @colemanw can you migrate this to 5.48.alpha1.mysql.tpl pls

File: CRM/Upgrade/Incremental/php/FiveFortySeven.php [hunk 2]
```diff
@@ -83,6 +83,16 @@ public function upgrade_5_47_alpha1($rev): void {
     }
   }
 
+  /**
+   * Upgrade step; adds tasks including 'runSql'.
+   *
+   * @param string $rev
+   *   The version number matching this function name
+   */
+  public function upgrade_5_47_1($rev): void {
```
Comment (seamuslee001) [inline CRM/Upgrade/Incremental/php/FiveFortySeven.php:92]: @colemanw I would remove this

Comment (colemanw) [reply CRM/Upgrade/Incremental/php/FiveFortySeven.php:92]: @seamuslee001 [the recommendation](https://github.com/civicrm/civicrm-core/pull/22893#issuecomment-1060071181) from @totten was to leave it in 5.47.1

Comment (totten) [reply CRM/Upgrade/Incremental/php/FiveFortySeven.php:92]: If the step is safe to re-run, then do move it.

If it's not safe to re-run... then... we don't *have* to move it. We put scary language like https://download.civicrm.org/latest/ in anticipation of these edge-cases.

I think if the time-frame to have affected sites is small (like 1 day of exposure; and only on RC sites), then I don't fell much guilt about doing it the lazy way. But if we're looking at, say, 4 days... then I'd push harder for a safer variant.

Comment (colemanw) [reply CRM/Upgrade/Incremental/php/FiveFortySeven.php:92]: K I'll move it. We also have to run Regen.sh on this one.

**Comment by demeritcowboy** [comment]
Is this waiting on something still? It runs ok for me.

Not a blocker but if the reasoning for moving it is to catch people in the last 2 days who've already installed 5.48 beta1, then my understanding is it should go in a function called beta2. But historically civi doesn't follow that pattern much, and it's only 2 days, and there's an understanding that they live on the edge if they install such things.

**Pull request merged**